lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/releases/graalvm/GraalCEReleaseProvider.scala

```diff
@@ -20,17 +20,22 @@ class GraalCEReleaseProvider(releaseProvider: SimpleReleaseProvider)
     extends GraalVMRuntimeReleaseProvider {
 
   /** @inheritdoc */
-  override def packageFileName(version: GraalVMVersion): String =
+  override def packageFileName(version: GraalVMVersion): String = {
     GraalCEReleaseProvider.packageFileNameForCurrentOS(version)
+  }
 
   /** @inheritdoc */
   override def downloadPackage(
     version: GraalVMVersion,
     destination: Path
   ): TaskProgress[Unit] = {
-    val tagName     = s"vm-${version.graalVersion}"
     val packageName = packageFileName(version)
-    val release     = releaseProvider.releaseForTag(tagName)
+    val tagName = if (version.graalMajorVersion < 23) {
+      s"vm-${version.graalVersion}"
+    } else {
+      s"jdk-${version.javaVersion}"
+    }
+    val release = releaseProvider.releaseForTag(tagName)
```

Comment:
We now have a new codepath that depends on Graal major version being less or greater than `23`.

But our unit tests only check Graal versions `1.0.0` and `2.0.0` both with "Java 11". Please add a test that has Graal major version >= 23 and uses the new Java format, e.g. `17.0.7` or something.

Reply:
Added in 93f504c085114ca26782f38fb670303cf6d137f7